package.json

```diff
@@ -49,7 +41,7 @@
     "lint-staged": "^7.0.0",
     "lodash": "^4.17.4",
     "merge2": "^1.2.1",
-    "prettier": "^1.10.2",
+    "prettier": "^1.13.7",
```

Comment:
You don't need this right? its shipped as part of `eslint-config-skyscanner-with-prettier`?

Reply:
Ehm yeah that's a good point

Reply:
Also just noticed that prettier should probably be removed from the `backpack-android` deps too...